code/modules/mob/living/simple_animal/hostile/netherworld.dm

```diff
@@ -0,0 +1,40 @@
+/mob/living/simple_animal/hostile/netherworld
+	name = "creature"
+	desc = "A sanity-destroying otherthing from the netherworlds."
+	icon_state = "otherthing"
+	icon_living = "otherthing"
+	icon_dead = "otherthing-dead"
+	health = 80
+	maxHealth = 80
+	obj_damage = 100
+	melee_damage_lower = 25
+	melee_damage_upper = 50
+	attacktext = "chomps"
+	attack_sound = 'sound/weapons/bite.ogg'
+	faction = list("creature")
+	speak_emote = list("screams")
+	gold_core_spawnable = 1
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	minbodytemp = 0
+
+/mob/living/simple_animal/hostile/netherworld/migo
+	name = "mi-go"
+	desc = "A pinkish, fungoid crustacean-like creature with numerous pairs of clawed appendages and a head covered with waving antennae."
+	icon_state = "mi-go"
+	icon_living = "mi-go"
+	icon_dead = "mi-go-dead"
+	attacktext = "lacerates"
+	var/list/migo_sounds
+
+/mob/living/simple_animal/hostile/netherworld/migo/Initialize()
+    . = ..()
+    migo_sounds = world.file2list("strings/migo_sounds.txt")
```

Comment:
dont store file paths in strings, they need to be in dm ```'single quotes'``` so if they're removed they throw errors, rather than silently waiting to runtime one day.

Reply:
'strings/migo_sounds.txt'

like that?

Reply:
No.
I mean you can't use a text file.
you have to use a list.

Reply:
do you know how long this list will be mam?

Reply:
don't give a poop, long list is infinitely better than a list that could literally runtime at any moment.

Reply:
Would `'[soundpath]'` not work for selecting sounds? I recall having done something like that similarly before.

Reply:
No it would not.
It's a compile time thing.

Feel free to keep ignoring me here @Armhulen it's not my fault when your code breaks.